addon/components/fd-sheets/fd-sheets-tool-bar.js

```diff
@@ -364,8 +348,15 @@ export default Component.extend(FdReadonlyProjectMixin, FdShareFunctionMixin, {
 
       @method actions.customButtonAction
     */
-    customButtonAction() {
-      this.get('customButtonSheet')();
+    customButtonAction(action, event) {
+      let actionType = typeof action;
+      if (actionType === 'function') {
+        action(event);
+      } else if (actionType === 'string') {
+        this.sendAction('customButtonAction', action);
```

Comment:
356:9  error  Use closure actions, unless you need bubbling  ember/closure-actions